### DISCUSSION TITLE
Game crashes when I tp to Lustia

### DISCUSSION THREAD

**Discussion by 69idklol** [ErinaSugino/Starbound-SBR-Mod#121] [Trouble Shooting]
Not every time, but sometimes the game crashes when I teleport to Lustia. I tried to remove all my mods trying to find out what causes the issue, but it still crashes even though the only mod I have currently is sexbound.
[starbound.log](https://github.com/user-attachments/files/15916034/starbound.log)

**Comment by ErinaSugino** [comment]
The log contains no errors and seems to end right on the teleport.
Since you say the game fully crashes, I would then need to crash message.

**Comment by 69idklol** [comment]
Not sure if that's it, but I got this from an earlier log, when the same thing happened.
 [22:18:56.617] [Error] Fatal Error: Stack overflow encountered

**Reply by ErinaSugino** [reply]
Wow, that's... mighty helpful as always, Starbound.
If that is literally the only thing related to this error, then I'm afraid I can't help. Between the only error message being "Something went wrong, but we don't know what" and it only occasionally happening? No clue.

**Reply by 69idklol** [reply]
Might be a dumb idea, but could it be related to audio issues? Because when I enter Lustia the background music is like glitched out and it sounds like there are multiple background musics playing at the same time, until I turn it off manually at the speakers.

**Reply by ErinaSugino** [reply]
That definitely sounds like something is wrong, but still doesn't tell us anything about what's going wrong and why.

**Reply by red3dred** [reply]
I've seen the issue mentioned by other users but it was because of lacking KawaiiStuff+.
There maybe an issue in Starbound itself, if it tries to log too many errors at once (Missing clothes) it might return this?
Anyhow, for that person, downloading KawaiiStuff+ fixed their problem. It's worth a shot.

**Reply by ErinaSugino** [reply]
But I included a raw copy of all the used clothes for the NPCs since SBR 1.2, so that KawaiiStuff+ isn't needed anymore...?

**Reply by red3dred** [reply]
> But I included a raw copy of all the used clothes for the NPCs since SBR 1.2, so that KawaiiStuff+ isn't needed anymore...?

Well then i really haven't the slightest on what this issue could possibly be. I know Lustia is somehow very overwhelming for most computers, but even then, this is some next level stuff if it Stack-overflows with barely any mods. We'll have to wait till we get the users to report it more times, so we'll be able to eventually recreate it, or i don't know, make Lustia less intensive somehow.

**Reply by 69idklol** [reply]
> I've seen the issue mentioned by other users but it was because of lacking KawaiiStuff+. There maybe an issue in Starbound itself, if it tries to log too many errors at once (Missing clothes) it might return this? Anyhow, for that person, downloading KawaiiStuff+ fixed their problem. It's worth a shot.

Now that you mention it I did install Kawaii+ because that was another problem, when I downloaded sbr for the first time it always crashed for me when I tried to tp to Lustia not just occasionally and even in the logs it were saying that something is missing from Kawaii+, so I downloaded it, since that it only crashes sometimes.
